CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native m1 support to the macOS platform. 
```

Comment:
"brings native m1 / Apple Silicon support to the macOS platform. It also represents a comprehensive updating of every third-party dependency."

Reply:
Sorry I have to amend this feedback: we did not update every single library dependency. Just a lot or most of them, as was needed for the M1 native build.

Reply:
It was a comprehensive review and update. Not everything was changed. Still a review.